README.md

```diff
@@ -90,6 +90,33 @@ D.get('year', date); // 2015
 
 ```
 
+###fromEpoch
```

Comment:
The code is perfect, but the name `fromEpoch` sounds to me like It would return a Date rather than take it as input.
Can we call it convertTo, or something similar ?